### DISCUSSION TITLE
Any PrismaStudio-like but with web access like most popular Headless CMS solutions out there?

### DISCUSSION THREAD

**Discussion by denik1981** [prisma/prisma#10938] [Q&A] [answered]
Any PrismaStudio-like solution but with web access, or any headless CMS with a built-in visual editor that can be configured to use Prisma graphql endpoints as its source DB? I have researched a lot of HeadlessCMS solutions out there and none gave me the ability to hook Prisma as the DB engine of those systems. In some headless CMS, like sanity for eg, you don't even own your DB, but when you do, you cannot connect to a graphql server. It is always straight to the DB and a few of them. And while this might look like no issue to some, the problem is that what you gain with the abstraction Prisma gave to you independent of the DB you want to use, you then lose all these benefits by having a direct relationship between the underlying DB (which could also be multiple) and the headless CMS.

Prisma Studio seems to be a kind of solution in there but it is nowhere close to what the visual editors of the most trendy headless CMS solutions look like and unfortunately, is desktop-based only. So it seems to me, it's was mostly designed to serve as a tool for the developer but not to the end-user.

Any ideas, alternatives, comments about this?

**Comment by jbruxelle** [comment, marked as answer]
Hi, did you try KeystoneJS ? It might correspond to what you are looking for... (it is based on Prisma)

**Comment by denik1981** [comment]
Yes, I think keystone.js is what I'm looking for.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
